.cargo/config.toml

```diff
@@ -1,4 +1,4 @@
-[build]
+[target.'cfg(all())']
```

Comment:
I think I tried this before and it didn't work. Did you verify it applies correctly?

Reply:
This was me fixing this before: https://github.com/vectordotdev/vector/pull/11674

Reply:
Ignore me, I see this has extra `()` now 😄